pom.xml

```diff
@@ -53,7 +53,7 @@
         <module>xtable-utilities</module>
         <module>xtable-aws</module>
         <module>xtable-hive-metastore</module>
-        <module>xtable-service</module>
+<!--        <module>xtable-service</module>-->
```

Comment:
This should be added back, any reason why you had to comment this out?

Reply:
it has been added back.